configs/inspection/Idealista.xml

```diff
@@ -77,7 +74,7 @@
   <inspection_tool class="ConstantNamingConvention" enabled="true" level="ERROR" enabled_by_default="true">
     <option name="onlyCheckImmutables" value="false" />
     <option name="m_regex" value="[A-Z][A-Z_\d]*" />
-    <option name="m_minLength" value="3" />
+    <option name="m_minLength" value="5" />
```

Comment:
Mmmm seems rare, I don't know why this value at this moment, I can undo it if you want, a TAG constant name length seems good to me